tests/test_maintel_enable_mtcs.py

```diff
@@ -0,0 +1,108 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+import random
+import unittest
+import asynctest
+
+from lsst.ts import salobj
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel import EnableMTCS
+from lsst.ts.observatory.control.mock import MTCSMock
+
+random.seed(47)  # for set_random_lsst_dds_domain
+
+logging.basicConfig()
+
+
+class TestEnableMTCS(standardscripts.BaseScriptTestCase, asynctest.TestCase):
+    async def basic_make_script(self, index):
+        self.script = EnableMTCS(index=index)
+        self.mtcs_mock = MTCSMock()
+
+        return (self.script, self.mtcs_mock)
+
+    async def test_run(self):
+        async with self.make_script():
+            await self.configure_script()
+
+            await self.run_script()
+
+            for comp in self.script.group.components:
+                if getattr(self.script.group.check, comp):
+                    with self.subTest(f"{comp} summary state", comp=comp):
+                        self.assertEqual(
+                            getattr(
+                                self.mtcs_mock.controllers, comp
+                            ).evt_summaryState.data.summaryState,
+                            salobj.State.ENABLED,
+                        )
+
+    async def test_ignore_feature(self):
+        async with self.make_script():
+
+            # Test the ignore feature
+
```

Comment:
Please consider deleting that comment -- the test is called `test_ignore_feature` so I think it adds no new information.